t/mojo/util.t

```diff
@@ -661,4 +661,31 @@ subtest 'Hide DATA usage from error messages' => sub {
   unlike $@, qr/DATA/, 'DATA has been hidden';
 };
 
+subtest 'urandom_token' => sub {
+  isnt urandom_token, urandom_token, "two tokens are not the same";
+  like urandom_token,                         qr/^[A-Za-z0-9]{44}$/,   "44 alphanumeric chars (default, 256 bits)";
+  like urandom_token(16),                     qr/^[A-Za-z0-9]{16}$/,   "16 alphanumeric chars";
+  like urandom_token(4096),                   qr/^[A-Za-z0-9]{4096}$/, "4096 alphanumeric chars";
+  like urandom_token(128, [qw/⚀ ⚁ ⚂ ⚃ ⚄ ⚅/]), qr/^[⚀⚁⚂⚃⚄⚅]{128}$/,     "128 custom alphabet chars";
+
+  subtest "mod bias" => sub {
+    my $len       = 2_000_000;
+    my @alphabet  = ("A" .. "Z", "a" .. "z", "0" .. "9");
+    my $tolerance = 0.05;
+
+    my %values;
+    foreach my $char (split "", urandom_token($len, \@alphabet)) {
+      $values{$char}++;
+    }
+
+    my $avg = $len / @alphabet;
+    foreach my $char (@alphabet) {
+      my $cnt  = $values{$char} || 0;
+      my $diff = abs($avg - $cnt);
+      cmp_ok($diff, "<", $tolerance * $avg, "$char is within tolerance");
```

Comment:
I don't know how reliable this test is, of course it is good to test the randomness but I am concerned about randomness tests causing potential false negatives. If appropriate it could be skipped unless an environment variable is present.

Reply:
Yea, this seems like it would be trouble in the future.

Reply:
This test has been removed as it's no longer needed